app/screens/more_channels/more_channels.js

```diff
@@ -340,6 +361,19 @@ export default class MoreChannels extends PureComponent {
                 }),
             };
 
+            let activeChannels;
+
+            switch (this.state.typeOfChannels) {
+            case 'public':
+                activeChannels = channels;
+                break;
+            case 'archived':
+                activeChannels = archivedChannels;
+                break;
+            default:
+                activeChannels = channels;
```

Comment:
shouldn't this be the option for `all channels` meaning it should be `channels` and `archivedChannels` together?

Reply:
So I've since changed this logic, but generally there shouldn't be any need for viewing all channels here, since we're viewing either/or public/archived channels.

Reply:
hmm, looking at the invision I see this
![image](https://user-images.githubusercontent.com/6757047/68088721-8a1c8e00-fe40-11e9-870e-1a08c505deb3.png)

@asaadmahmood can you confirm that in the mobile app we do not have an `All channels` option?